smokes/e2e/login.scenarios.coffee

```diff
@@ -0,0 +1,60 @@
+# coffee script
+# test goal: checks the capability to identify the user and to start a build
+# according the master.cfg file and authorization part
+
+homePage = require('./pages/home.coffee')
+forcePage = require('./pages/force.coffee')
+builderPage = require('./pages/builder.coffee')
+
+describe 'identify user as different role and check the behavior', () ->
+    home = null
+    force = null
+    builder = null
+
+    beforeEach () ->
+        home = new homePage()
+        builder = new builderPage('runtests', 'force')
+        force =  new forcePage()
+
+    afterEach () ->
+        home.setLogOut()
+
+    # DONT UNCOMMENT ISSUE present on buildbot 9 when the creds are not correct
+    # Test to reactivate when the issue will be not more present
+
+    # it 'try to start a build without be identified', () ->
```

Comment:
I'd rather un comment the test, but disable it (by using ``xit`` instead of ``it``)

https://jasmine.github.io/2.1/introduction.html#section-Pending_Specs